src/main/java/com/iridium/iridiumskyblock/shop/ShopManager.java

```diff
@@ -116,9 +116,13 @@ public void buy(Player player, ShopItem shopItem, int amount) {
 
             ItemStack itemStack = shopItem.type.parseItem();
             itemStack.setAmount(amount);
-            ItemMeta itemMeta = itemStack.getItemMeta();
-            itemMeta.setDisplayName(StringUtils.color(shopItem.name));
-            itemStack.setItemMeta(itemMeta);
+            ItemMeta itemMeta   = itemStack.getItemMeta();
+
+            if (shopItem.displayName != null) {
+                itemMeta.setDisplayName(StringUtils.color(shopItem.displayName));
+                itemStack.setItemMeta(itemMeta);
+                player.getInventory().addItem(itemStack);
+            }
```

Comment:
I think your logic is a little bit off, in addition to the intentation.
```suggestion
            if (shopItem.displayName != null && !shopItem.displayName.isEmpty()) {
                ItemMeta itemMeta = itemStack.getItemMeta();
                itemMeta.setDisplayName(StringUtils.color(shopItem.displayName));
                itemStack.setItemMeta(itemMeta);
            }
            
            player.getInventory().addItem(itemStack);
```

Reply:
ye u must check strings is not always null